tests/README.md

```diff
@@ -0,0 +1,63 @@
+# Tests
+
+These tests should ideally be run for every Pull Request, though due to the need to run `clasp login` before some commands, some tests are not run.
+
+
+# Configuration using Travis
```

Comment:
Only 1 h1 per doc. Make this an h2: `##`.

Reply:
👍